client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -914,6 +916,23 @@ def get_template_preset(self):
             "create_first_version": create_first_version
         }
 
+    def resolve_template_path(self, path: str) -> str:
```

Comment:
Should this contain the formatting?
```python
result = StringTemplate.format_template(path, fill_data)
if result.solved:
    path = result.normalized()
```

Reply:
`resolve_template_path` is actually just an additional step to support some DCC specific keys.
from my testing in Houdini, it's helpful to have it after the AYON's template resolving.
here's how it's used un Houdini: 

https://github.com/fabiaserra/ayon-houdini/blob/c5784bb456840ccc4bc583fdd4f81d83521abf38/client/ayon_houdini/api/workfile_template_builder.py#L24-L33

maybe we can refactor the method name to reflect that ?

Reply:
Actually - I love that @iLLiCiTiT because that also allows to do tweaks BEFORE that template formatting.

Reply:
Implemented with https://github.com/ynput/ayon-core/pull/875/commits/a8a69766ad00ea56b51f5e0c8dd3bdffebc1c67d